cli/testdataseeder.php

```diff
@@ -1,28 +1,28 @@
 <?php
 define('CLI_SCRIPT', 1);
-include('../../../../../../config.php');
+require(__DIR__.'/../../../../../../config.php');
```

Comment:
pls use require_once

Reply:
This came directly from the Moodle Docs. Do we need to change it?

>CLI_SCRIPT constant
All scripts must declare themselves as CLI scripts properly by setting the CLI_SCRIPT constant before loading the main config.php file.
>
>```
>define('CLI_SCRIPT', true);
>require(__DIR__.'/../../../config.php');
>```
>
>The path to the config.php must not use relative paths starting with ../. The best practise is to avoid dirname() function call and use the __DIR__ magic constant (this applies everywhere, not just for CLI scripts).

https://docs.moodle.org/dev/CLI_scripts#CLI_SCRIPT_constant

Reply:
I see. I don't really like simple require because it can cause problems but at this case that's fine.